tests/manage/pv_services/test_pvc_resize.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.framework.testlib import skipif_ocs_version, ManageTest, tier1
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@skipif_ocs_version('<4.5')
+@pytest.mark.parametrize(
+    argnames=["interface"],
+    argvalues=[
+        pytest.param(
+            constants.CEPHBLOCKPOOL, marks=pytest.mark.polarion_id("OCS-325")
+        ),
+        pytest.param(
+            constants.CEPHFILESYSTEM, marks=pytest.mark.polarion_id("OCS-325")
+        )
+    ]
+)
+class TestPvcResize(ManageTest):
+    """
+    Tests to verify PVC resize
+
+    """
+    @pytest.fixture(autouse=True)
+    def test_setup(self, interface, storageclass_factory, pvc_factory, pod_factory):
+        """
+        Create resources for the test
+
+        """
+        self.sc_obj = storageclass_factory(interface=interface)
+        self.pvc_obj = pvc_factory(
+            interface=interface, storageclass=self.sc_obj, size=3
+        )
+        self.pod_obj = pod_factory(interface=interface, pvc=self.pvc_obj)
+
+    def test_pvc_resize(self):
+        """
+        Verify PVC resize feature
+
+        """
+        pvc_size_new = 8
+
+        # Modify size of PVC and verify the change
+        self.pvc_obj.resize_pvc(pvc_size_new, True)
+
+        # Wait for 60 seconds to reflect the change on pod
```

Comment:
the 60 seconds needs to be changed.
Unless the timeout bellow needs to be changed to 60

Reply:
Ah. sure , missed to update the source comment while increasing the timeout. updated.